config/duration.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/0xERR0R/blocky/log"
+	"github.com/hako/durafmt"
+	"golang.org/x/exp/constraints"
+)
+
+type Duration struct{ time.Duration }
+
+func NewDuration[T constraints.Integer | time.Duration](value T) Duration {
+	return Duration{Duration: time.Duration(value)}
+}
+
+func (c Duration) Cast() time.Duration {
```

Comment:
👍  `cfg.someProperty.Cast()` is definitively more readable as `time.Duration(cfg.someProperty)`

Suggestion: maybe we should rename "Cast" to "asDuration" or "toDuration"?

Reply:
Agreed `ToDuration` would be more readable as the name implies the result.

Reply:
Went with `ToDuration` :)